docs/training.rst

```diff
@@ -109,7 +109,7 @@ Identifying Misrecognized Elements
 .. note::
    Make sure you have the latest trained coefficients and biases pasted into your ruleset before you do this, or recognition won't work well.
 
-The Evaluator can also point out misrecognized elements, in case the tag exerpts emitted by the trainer are insufficient to identify them. When you click Evaluate, as above, any pages with misrecognized nodes will show up in red; click those to see which element was wrongly selected. Unfortunately, you need to manually show the dev tools and switch to the Fathom panel once you get to the page in question; there aren’t yet web extension APIs to do it automatically. Once you do, you’ll see a quick and dirty representation of the “bad” element: a new label called “BAD [the trainee ID]”. Be sure to delete this if you choose to re-save the page for some reason. Also note that the BAD label is created only when the bad cell is clicked, for speed; if you navigate to the bad page manually, the label won’t be there, or there might be an old label from a previous iteration.
+FathomFox's Evaluator can point out misrecognized elements, in case the tag exerpts emitted by the trainer are insufficient to identify them. To start the process of using the Evaluator, open an instance of Firefox with FathomFox and your ruleset loaded (:doc:`fathom-fox<commands/fox>` makes this simple), and then open all of the samples you want to diagnose as separate tabs. Next, open the Evaluator page using the FathomFox's browser action button. The Evaluator page is initially quite empty: You have a dropdown to select your trainee ID and a button to run the Evaluator. When you click Evaluate, any pages with misrecognized nodes will show up in red; click those to see which element was wrongly selected. Unfortunately, you need to manually show the dev tools and switch to the Fathom panel once you get to the page in question; there aren’t yet web extension APIs to do it automatically. Once you do, you’ll see a quick and dirty representation of the “bad” element: a new label called “BAD [the trainee ID]”. Be sure to delete this if you choose to re-save the page for some reason. Also note that the BAD label is created only when the bad cell is clicked, for speed; if you navigate to the bad page manually, the label won’t be there, or there might be an old label from a previous iteration.
```

Comment:
~~start the process of using~~  -> "use"

Reply:
We might as well just inline the whole relevant syntax for fathom-fox: `fathom-fox -r ruleset.js`.

Reply:
Yes, this is kind of a big block of text, especially to keep your place in while trying to follow its instructions. How about an ordered list?